build.gradle

```diff
@@ -25,10 +25,13 @@ dependencies {
 	modImplementation "net.fabricmc.fabric-api:fabric-api:${project.fabric_version}"
 
 	modImplementation 'xyz.nucleoid:plasmid:0.5+1.18.2-SNAPSHOT'
-	modImplementation include('xyz.nucleoid:more-codecs:0.1.5')
+	modImplementation include('xyz.nucleoid:more-codecs:0.2.2+1.18')
 
 	modCompileOnly "maven.modrinth:styled-chat:1.2.0"
 	modCompileOnly "com.viaversion:viaversion-api:4.3.1"
+	modCompileOnly "dev.gegy:player-roles-api:1.5.2"
+	// Depend on the full mod to use internals that aren't in the API
+	modCompileOnly "dev.gegy:player-roles:1.5.2"
```

Comment:
Maybe these should be in the API? 😄

Reply:
Yeah some of the impls for things should probably be moved into the API, however extras (along with the rest of the libraries and minigames) are still on 1.18, but player-roles has already moved onto 1.19, and so I wasn't sure if this would then have to wait everything updates to be able to use the a newer player-roles version

Reply:
It seems like [`RoleOverrideMap`](https://github.com/NucleoidMC/player-roles/blob/aa6ab077e27165b03921d36f0f3a580d2e986ab5/src/main/java/dev/gegy/roles/override/RoleOverrideMap.java) could easily be replaced with a custom [`RoleOverrideReader`](https://github.com/NucleoidMC/player-roles/blob/aa6ab077e27165b03921d36f0f3a580d2e986ab5/api/src/main/java/dev/gegy/roles/api/override/RoleOverrideReader.java) implementation, since I don't believe the overrides need to be modified.

Reply:
Ah I missed that a lot of the `RoleOverrideMap` methods have `default` impls, and as of the new commit it should now just be dependent on player-roles-api